pkg/option/config.go

```diff
@@ -3279,6 +3283,19 @@ func (c *DaemonConfig) Populate(vp *viper.Viper) {
 			ToFQDNsMaxDeferredConnectionDeletes)
 	}
 	switch {
+	case vp.IsSet(ToFQDNsActiveConnectionsTTL): // set by user
+		if vp.GetInt(ToFQDNsActiveConnectionsTTL) >= int(2*defaults.DNSGCJobInterval) {
+			c.ToFQDNsActiveConnectionsTTL = vp.GetInt(ToFQDNsActiveConnectionsTTL)
+		} else {
+			log.Infof("Using the default value for %s: %d as user value %d is smaller than the default one", ToFQDNsActiveConnectionsTTL, defaults.ToFQDNsActiveConnectionsTTL, defaults.ToFQDNsActiveConnectionsTTL)
+			c.ToFQDNsActiveConnectionsTTL = int(defaults.ToFQDNsActiveConnectionsTTL)
+		}
+
+	default:
+		log.Infof("Using default value for %s: %d", ToFQDNsActiveConnectionsTTL, defaults.ToFQDNsActiveConnectionsTTL)
```

Comment:
If you add the flag to the daemon, then the daemon will already print out the configuration for this flag in a standardized way, so then there would be no need to print it here.
```suggestion
```